adoc/bp_postgresql_database_migration.adoc

```diff
@@ -39,44 +40,43 @@ smdba-1.5.8-0.2.3.1.x86_64
 
 [IMPORTANT]
 ====
-Postgresql 9.6 requires smdba version 1.5.8 or higher
+PostgreSQL 9.6 requires smdba version 1.5.8 or higher
 ====
 
 [WARNING]
 ====
 Always create a database backup before performing a migration
 ====
 
-
-$$.$$
 The database migration begins by executing the following command:
 
 ----
 $> /usr/lib/susemanager/bin/pg-migrate.sh
 ----
 
 
-The [path]``pg-migrate.sh``
- script will automatically perform the following operations:
+The [path]``pg-migrate.sh`` script will automatically perform the following operations:
```

Comment:
I do not have the context here but want to point out that meanwhile we are using new scripts that explicitly contain the version (pg-migrate-94-to-96.sh and pg-migrate-96-to-10.sh). For compatibility, the old name pg-migrate.sh gets installed as symbolic link to pg-migrate-94-to-96.sh

As I already said, I would like to get rid of any mention of version 9.4 at all. This database version cannot any longer be used since about a year as it is missing new functionality we are using meanwhile.

Reply:
I'm referencing the new script in the section about the 9.6 to 10 migration.  But you are right, there was a error with the sectioning levels...

I see your point that 9.4 is very outdated.  But in the description of https://github.com/SUSE/spacewalk/issues/8324 (from July 2019) we have a report that someone was still using 9.4 and now wanted to upgrade SLES 12 SP4 (with postgresql 10).  That why I think we should still mention 9.4 in the SUMA 3.2 documentation.  We no longer provide this migration description (from 9.4 to 9.6) in the SUMA 4 documentation.

Thanks for your help, Hubert!